components/brave_new_tab_ui/components/default/cryptoDotCom/style.ts

```diff
@@ -51,194 +55,207 @@ interface StyleProps {
-  ${(p) =>
-    p.$mr &&
-    `margin-right: ${typeof p.$mr === 'number' ? `${p.$mr}px` : p.$mr}`}
-  ${(p) =>
-    p.$mb &&
-    `margin-bottom: ${typeof p.$mb === 'number' ? `${p.$mb}px` : p.$mb}`}
-  ${(p) =>
-    p.$ml &&
-    `margin-left: ${typeof p.$ml === 'number' ? `${p.$ml}px` : p.$ml}`}
- `
+    ${getBoxStyle}
+`
 
 export const Box = styled(BasicBox)<StyleProps>`
   border: 1px solid rgba(79, 86, 97, 0.7);
-  padding: ${p => (p.hasPadding ? '0.5em' : '0')};
   border-radius: 2px;
-  display: ${p => (p.isFlex ? 'flex' : 'block')};
+  ${p => p.hasBottomBorder === false && 'border-bottom: none;'}
+  ${p => (!p.$p && p.$p !== 0) && 'padding: 0.5em;'}
   height: ${p => (p.$height ? `${p.$height}px` : 'auto')};
-
-   ${(p) =>
-     p.isFlex &&
-     `
-      flex-direction: ${p.column ? 'column' : 'row'};
-      justify-content: space-between;
-      align-items: center;
-    `}
- `
+`
 
 export const FlexItem = styled('div')<StyleProps>`
   display: ${p => (p.isFlex ? 'flex' : 'block')};
   flex: ${p => p.flex || 'inherit'};
   text-align: ${p => p.textAlign || 'left'};
   padding: ${p => (p.hasPadding ? '0.5em' : '0px')};
-  width: ${p => (p.isFullWidth ? '100%' : 'auto')};
   border-bottom: ${p => (p.hasPadding ? '1px solid rgba(79, 86, 97, 0.7)' : 'none')};
 
   ${(p) =>
     p.isFlex &&
     `
-     flex-direction: ${p.column ? 'column' : 'row'};
-     justify-content: space-between;
-     align-items: center;
-   `}
+    flex-direction: ${p.column ? 'column' : 'row'};
+    justify-content: space-between;
+    align-items: center;
+  `}
 
-  ${(p) =>
-    p.$pl &&
-    `padding-left: ${typeof p.$pl === 'number' ? `${p.$pl}px` : p.$pl}`}
-  ${(p) =>
-    p.$pr &&
-    `padding-right: ${typeof p.$pr === 'number' ? `${p.$pr}px` : p.$pr}`}
- `
+  ${getBoxStyle}
+`
+
+export const ButtonGroup = styled(BasicBox)<StyleProps>`
+  border: 1px solid rgba(255, 255, 255, 0.2);
+  border-radius: 2px;
+  display: flex;
+  justify-content: center;
+  width: max-content;
+  margin: 11px auto;
+`
 
 export const PlainButton = styled('button')<StyleProps>`
   display: ${p => p.inline ? 'inline-block' : 'block'};
   background: none;
   border: none;
-  cursor: pointer;
+  cursor: ${p => p.disabled ? 'auto' : 'pointer'};
   color: ${p => getColor(p.textColor) || '#ffffff'};
 
-  &:focus {
-    outline: 0;
+  &:focus:not(:focus-visible) {
+    outline: none;
   }
 
-  ${(p) =>
-    p.$m &&
-    `margin: ${typeof p.$m === 'number' ? `${p.$m}px` : p.$m}`}
-  ${(p) =>
-    p.$mt &&
-    `margin-top: ${typeof p.$mt === 'number' ? `${p.$mt}px` : p.$mt}`}
-  ${(p) =>
-    p.$mr &&
-    `margin-right: ${typeof p.$mr === 'number' ? `${p.$mr}px` : p.$mr}`}
-  ${(p) =>
-    p.$mb &&
-    `margin-bottom: ${typeof p.$mb === 'number' ? `${p.$mb}px` : p.$mb}`}
-  ${(p) =>
-    p.$ml &&
-    `margin-left: ${typeof p.$ml === 'number' ? `${p.$ml}px` : p.$ml}`}
- `
+  ${getTextStyle}
+
+  ${getBoxStyle}
+
+  ${ButtonGroup} & {
+    color: ${p => p.isActive
```

Comment:
@petemill Needs help here.
After rebasing, `p.isActive` is always `undefined` even if this is set via props like this - https://github.com/brave/brave-core/pull/7699/files#diff-aedb3878e9005617ae3c040c085d7c37b004695828835d3c689ddfd9b61e8a99R53

Reply:
Avoided this problem by using new `inButtonGroup` prop - https://github.com/brave/brave-core/pull/7699/commits/d7d20938d48e754e1b0f9e045318a2f1c471ee41